src/base-components/switch-base/switch-base.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import './switch-base.css'
+
+const _switchClasses = () => ({
+  checkbox: 'w-0 h-0 hidden',
+  disabled: 'opacity-50 cursor-not-allowed',
+})
+
+const SwitchBase = ({ id, classes, styles, checked, onChange, disabled, tabIndex }) => {
+  const switchClasses = _switchClasses()
```

Comment:
see https://github.com/EQWorks/lumen-labs/pull/6#discussion_r657373998 for the button PR and see if we can use a more static approach to reduce memory activity while using these components